jms/src/main/scala/akka/stream/alpakka/jms/Jms.scala

```diff
@@ -71,6 +76,7 @@ final case class JmsConsumerSettings(connectionFactory: ConnectionFactory,
                                      bufferSize: Int = 100,
                                      selector: Option[String] = None,
                                      acknowledgeMode: Option[AcknowledgeMode] = None,
+                                     ackTimeout: Duration = Duration.Inf,
```

Comment:
Wouldn't it be better to have a finite default? The uncommitted futures might pile up.

Reply:
You're right. I tried to retain the previous behavior. The question is what timeout makes sense? We're using a 1 second timeout for our use cases and it seems to be stable. Let me know if there is a rationale for a different value.